sdk/keyvault/azure-keyvault-keys/tests/test_key_client.py

```diff
@@ -422,12 +423,20 @@ def test_get_random_bytes(self, client, **kwargs):
 
         generated_random_bytes = []
         for i in range(5):
-            random_bytes = client.get_random_bytes(count=8)
+            random_bytes = client.get_random_bytes(count=8).value
```

Comment:
Consider reworking this test into an example we can pull into the method's docstring. Now that the method returns a model, I think we need to demonstrate getting the bytes off it.